docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:11
+
+RUN apt-get update \
+    && apt-get install -y time
+
+COPY logging.xml /root
+COPY *.jar /root/
+COPY *.sh /root/
+
+CMD ["sleep", "365d"]
```

Comment:
why sleep?

Reply:
Currently, all pods share the same image: the memory server, the relay server, and the client. This allows us to test different configurations. For instance, it allows us to run the memory and the relay either on the same pod or on separate pods.